crates/cli-support/src/js/closures.rs

```diff
@@ -261,10 +275,37 @@ impl ClosureDescriptors {
         }
         Ok(())
     }
+
+    /// The final step, rewriting calls to `__wbindgen_describe_closure` to the
+    /// imported functions
+    ///
+    /// Here we take the final step, rewriting calls to
+    /// `__wbindgen_describe_closure` to the imported functions, ensuring
+    /// there's no lingering references to `__wbindgen_describe_closure`.
+    fn rewrite_calls(&self, input: &mut Context) {
+        // FIXME: Ok so this is a bit sketchy in that it introduces overhead.
+        // What we're doing is taking a our #[inline(never)] shim and *not*
+        // removing it, only switching the one function that it calls internally.
+        //
+        // This isn't great because now we have this non-inlined function which
+        // would certainly benefit from getting inlined. It's a tiny function
+        // though and surrounded by allocation so it's probably not a huge
+        // problem in the long run.
```

Comment:
If we are going to make excuses for ourselves here, we might as well mention that `wasm-opt` will likely inline the function for us anyways.